app/code/community/Etailer/Backorders/Model/Reports/Resource/Product/Lowstock/Collection.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/**
+ * Magento
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/osl-3.0.php
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@magento.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade Magento to newer
+ * versions in the future. If you wish to customize Magento for your
+ * needs please refer to http://www.magento.com for more information.
+ *
+ * @category    Mage
+ * @package     Mage_Reports
+ * @copyright  Copyright (c) 2006-2014 X.commerce, Inc. (http://www.magento.com)
+ * @license    http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+
+
+/**
+ * Product Low Stock Report Collection
+ *
+ * @category    Mage
+ * @package     Mage_Reports
+ * @author      Magento Core Team <core@magentocommerce.com>
+ */
+class Etailer_Backorders_Model_Reports_Resource_Product_Lowstock_Collection extends Mage_Reports_Model_Resource_Product_Lowstock_Collection 
+{
+    /**
+     * Add Notify Stock Qty Condition to collection
+     *
+     * @param int $storeId
+     * @return Mage_Reports_Model_Resource_Product_Lowstock_Collection
+     */
+    public function useNotifyStockQtyFilter($storeId = null)
+    {
+      	Mage::log('My log entry');
```

Comment:
Sorry, forgot to remove that. You can remove.
On Jun 27, 2016 13:16, "Robert Coleman" <notifications@github.com> wrote:

> In
> app/code/community/Etailer/Backorders/Model/Reports/Resource/Product/Lowstock/Collection.php
> <https://github.com/etailer/Etailer_Backorders/pull/8#discussion_r68673440>
> :
>
> > + *
> > + * @category    Mage
> > + * @package     Mage_Reports
> > + * @author      Magento Core Team <core@magentocommerce.com>
> > + */
> > +class Etailer_Backorders_Model_Reports_Resource_Product_Lowstock_Collection extends Mage_Reports_Model_Resource_Product_Lowstock_Collection
> > +{
> > +    /**
> > +     * Add Notify Stock Qty Condition to collection
> > +     *
> > +     * @param int $storeId
> > +     * @return Mage_Reports_Model_Resource_Product_Lowstock_Collection
> > +     */
> > +    public function useNotifyStockQtyFilter($storeId = null)
> > +    {
> > +      	Mage::log('My log entry');
>
> why is this here?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/etailer/Etailer_Backorders/pull/8/files/378fb99a414b21b607e74f0a815898e82ac9a36d#r68673440>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AQaVeFrYVBQRRx1aAFvrCpoQxZX0IkWeks5qQFnWgaJpZM4I-NJe>
> .
>